backend/ops_api/ops/schemas/budget_line_items.py

```diff
@@ -106,7 +106,7 @@ def validate_agreement_reason_must_not_have_incumbent(self, data, **kwargs):
             if (
                 bli
                 and bli.agreement_id
-                and bli.agreement.agreement_reason == AgreementReason.NEW_REQ
```

Comment:
don't delete the check for NEW_REQ
include it before the isinstance chech

```
                bli
                and bli.agreement_id
                and bli.agreement.agreement_reason == AgreementReason.NEW_REQ
                and isinstance(bli.agreement, ContractAgreement)  # only contracts have incumbents
                and bli.agreement.incumbent_id
```

Reply:
thx @stevtek brought it back